tensorflow/lite/micro/tools/make/renode_download.sh

```diff
@@ -60,6 +60,26 @@ else
 
   # Get link to requested version
   RELEASES_JSON=`curl https://api.github.com/repos/renode/renode/releases 2>/dev/null`
+
+  # Make sure API rate limit is not exceeded and if so retry
+  RATE_LIMIT="API rate limit exceeded"
```

Comment:
The curl retry logic is complicated.

Let's use wget instead. Something like:
```
wget --tries=10 --timeout=1
```

And let's also link to the github issue so that it is easy to see that the retry logic was added for a reason.

Reply:
Well this seemingly gives the same behavior:
wget --retry-on-http-error=403 --tries=10 --timeout=1 -O - https://api.github.com/repos/renode/renode/releases 

However when testing, the script with curl and the wget line seems to behave differently.

With the script at least sometimes it succeed after a few retries (The normal case is that it succeed for both curl and wget on first attempt but you have to run it many times until getting the first 403 and then it comes often).
Whereas with the wget line, once it gets 403 on the first attempt it always keep getting 403 for all subsequent retries.
It's like the server is more curl friendly. Perhaps I am missing something or perhaps the server sees it as wget is connected all the time even when retrying whereas in the curl case, curl is really disconnecting and retrying.

Anyway sometimes also the script gets stuck with 403 after retrying when testing it heavily (although for normal usage I believe it would handle the few cases of 403) so it is not bulletproof.
However with the extracted download url, i.e. https://github.com/renode/renode/releases/download/v1.11.0/renode-1.11.0.linux-portable.tar.gz I haven't yet seen any download problem.
So a more robust solution would then probably be to fall back to this hard-coded url in case of 403 instead of extracting it from json. What do you think?

Reply:
Thanks for looking into this!

Instead of a fallback to the hard-coded url, let's keep things simple and always go with the hardcoded url + wget. This would also make it more consistent with the other TFLM downloads.

The value of knowing that there is a newer release is less than the maintenance overhead of a more complicated download setup.

Reply:
Ok

Reply:
Done